conda-reqs.yaml

```diff
@@ -23,11 +23,28 @@ dependencies:
     # bundle FireSim driver with deps into installer shell-script
     - constructor
 
+    # Lock down the CPP deps so future updates are tightly controlled
+    - ld_impl_linux-64==2.39
+    - libdeflate==1.14
+    - libtool==2.4.6
+    - lz4-c==1.9.3
+    - libnghttp2==1.47.0
+    - libprotobuf==3.21.11
+    - krb5==1.19.3
+    - libllvm15==15.0.6
+    - libtiff==4.4.0
+    - openssl==3.0.7
+    - pkginfo==1.9.2
+    - harfbuzz==5.3.0
+    - aws-sam-translator==1.55.0
+    - fontconfig==2.14.1
+
```

Comment:
conda-lock doesn't support updating one or two dependencies, without updating all others to the latest version. The solution of locking everything down came about after a discussion with @davidbiancolin : He said that the prebuilt drivers will break in future revisions if we have CPP dependencies constantly updating.

The reason I need to touch this at all, is so that I can add: 
    - fsspec, s3fs, aiohttp

Reply:
the CPP dependencies are going to be bundled with the driver at build time in the tarball, aren't they?

Reply:
I was worried about potential discrepancies between the two ways we build the driver. That may be ill-founded. We'll need to take this out-of-band.

Reply:
conda-lock does support 'updating' something that is in the existing lockfile to a new version in a minimal fashion with it's `--update` flag.  However, it seems like you can't add new packages to the requirements spec and also use `--update` for them to minimally install new packages.  I am trying to get a discussion of this usecase going in https://github.com/conda/conda-lock/pull/355#discussion_r1111114862

I think I see how I could patch conda-lock to support using `--update` with new packages.  I'm not sure how long it would take me to work through actually making that enhancement given that I just started looking at the project's code.  However, I'm hoping that this is just a documentation issue with conda-lock and that it's already possible to do this or trivial for the original dev or maintainers.